images/kairos-ubuntu-24-lts/Earthfile

```diff
@@ -0,0 +1,14 @@
+VERSION 0.8
+FROM alpine
+
+# Builds our base image that can be used standalone as well as for the Kairos Factory:
+# https://kairos.io/docs/reference/kairos-factory/
+base-image:
+    ARG TAG=dev
+    ARG TARGETPLATFORM
+    FROM DOCKERFILE \
+      -f Dockerfile \
+      --platform $TARGETPLATFORM \
```

Comment:
why can't we do this from within Earthly itself?

Reply:
We could, but I'd like to be able to run `hadolint` on our image build files, especially since we're talking about doing more with these images, and that can't check `Earthfile`s.